### DISCUSSION TITLE
Admin enviar mensaje de error y no se puede guardar la modificación generada

### DISCUSSION THREAD

**Discussion by nicolasdurans** [Cotalker/documentation#742] [4. Bug Reports] [answered]
![image](https://user-images.githubusercontent.com/5656833/157320501-a16b98b1-81f8-4b37-9a41-ef43defa8573.jpeg)

**Comment by acedward** [comment, marked as answer]
Se ve bien, /fid puede retornar 404 si no encuentra la question.

EDIT: issue creado

**Reply by acedward** [reply]
Hay otro endpoint que falla?

**Reply by nicolasdurans** [reply]
Aquí ejemplo del problema que le pasó a un cliente 

https://drive.google.com/file/d/1o23DpEtdxdbbFC31acPLEDdIMcBIsyoM/view

Minuto 34:00 al 35:00

**Reply by acedward** [reply]
Issue para mensaje de error
https://github.com/Cotalker/documentation/issues/744
